From 99c7fa0104a9b9efe49b726f47a9f5d2554a2d71 Mon Sep 17 00:00:00 2001
From: Sandor Yu <Sandor.yu@nxp.com>
Date: Fri, 17 Nov 2017 16:14:07 +0800
Subject: [PATCH 2826/5242] MLK-16886: hdp: Disable EDID function

commit  e4ea1280b4357d96c2ad15f785cc80d5b4554dc3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable EDID function temporary
because edid read is unstable on some boards.

Signed-off-by: Sandor Yu <Sandor.yu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdp.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdp.c b/drivers/gpu/drm/imx/hdp/imx-hdp.c
index 06ba4f27..145d86f 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdp.c
@@ -905,7 +905,7 @@ static int imx8qm_hdp_swrite(struct hdp_mem *mem, unsigned int addr, unsigned in
 };
 
 static struct hdp_devtype imx8mq_hdmi_devtype = {
-	.is_edid = true,
+	.is_edid = false,
 	.is_4kp60 = true,
 	.ops = &imx8mq_ops,
 	.rw = &imx8mq_rw,
-- 
1.7.9.5

